src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -988,6 +991,7 @@ def to_json(self) -> dict:
         if self.daemon_type == 'osd':
             out['osdspec_affinity'] = self.osdspec_affinity
         out['is_active'] = self.is_active
+        out['deployed_by'] = self.deployed_by
```

Comment:
is there a need to hide those things from the orchestrator cli? This really is an implementation detail

Reply:
It don't think it needs to be exposed to the cli. I'm pretty sure I originally put this here because I was thinking it was necessary for the field to be persistent similar to is_active but since deployed_by is actually set when refreshing hosts and daemons it's not necessary. I'll take it out of to_json.